### PR TITLE
Remove commented out code

### PR DESCRIPTION
**Pull request by ajesusflores**
## Description

Improve readability by removing commented out code.

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [ ] Please add REST spec PR link to the SDK PR
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [ ] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [ ] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [ ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [ ] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Review by dsgouda — APPROVED** [review]
LGTM

**Review by dsgouda — CHANGES_REQUESTED** [review]
@shahabhijeet please take a look, looks innocuous

File: src/SdkCommon/ClientRuntime.Azure/ClientRuntime.Azure/AzureClientExtensions.cs [hunk 1]
```diff
@@ -77,8 +77,6 @@ public static partial class AzureClientExtensions
             IAzureLRO<TBody, THeader> lro = ScheduleLRO<TBody, THeader>(client, response, customHeaders, cancellationToken);
             await lro.BeginLROAsync();
             return await lro.GetLROResults();
-
-            //return await LegacyLro<TBody, THeader>(client, response, customHeaders, cancellationToken);
```
Comment (shahabhijeet) [inline src/SdkCommon/ClientRuntime.Azure/ClientRuntime.Azure/AzureClientExtensions.cs]: @ajesusflores we recently updated our async operation handling code.
The reason the commented code was not removed is to have the legacy code ready to be used as a fallback during debugging.
Yes this is a bad practice and I realized it, but trust me this code is long overdue on cleaning up commented code.
We will clean up this code soon.
Thank you for your diligence.
I am closing this PR by end of this week.

**Review by shahabhijeet — CHANGES_REQUESTED** [review]

**Comment by dsgouda** [comment]
Per @shahabhijeet 's  comment we will dedicate some time in the future to clean up code in our ClientRuntime projects.
Thanks @ajesusflores for the feedback.
Closing this PR

**Pull request closed without merging**